x-pack/build.gradle

```diff
@@ -36,7 +36,9 @@ subprojects {
     additionalLicense 'ELAST', 'Elastic License 2.0', '2.0; you may not use this file except in compliance with the Elastic License'
   }
 
-  ext.projectLicenses = ['Elastic License 2.0': ext.elasticLicenseUrl]
+  project.getPluginManager().withPlugin("elasticsearch.publish") {
```

Comment:
Shouldn't this be conditional on the licensing plugin being applied?

Reply:
fixed